src/app/helptext/services/components/service-netdata.ts

```diff
@@ -0,0 +1,75 @@
+import { T } from '../../../translate-marker';
+import { Validators } from '@angular/forms';
+import {
+    regexValidator
+  } from '../../../pages/common/entity/entity-form/validators/regex-validation';
+
+export default {
+    global_paratext: T('Global Settings'),
+
+    history: {
+        placeholder: T('History'),
+        tooltip: T('The number of <samp>entries</samp> the netdata daemon will\
+ by default keep in memory for each chart dimension. Default is 86400.'),
+     validation: [ Validators.required, regexValidator(/^\d+$/) ]
+    },
+    update_every: {
+        placeholder : T('Update Frequency'),
+        tooltip: T('The frequency, in seconds, for data collection.'),
+        validation: [Validators.required, regexValidator(/^\d+$/) ]
+    },
+    http_port_listen_backlog: {
+        placeholder : T('HTTP Port Listen Backlog'),
+        tooltip: T('The port backlog'),
+        validation: [ Validators.required, regexValidator(/^\d+$/) ]
+    },
+    bind: {
+        placeholder : T('Bind to'),
+        tooltip: T('Select one or more IP addresses to which to bind the Netdata service.')
+    },
+    port: {
+        placeholder : T('Bind to port'),
+        tooltip: T('The port which will be used with selected bind to IP addresses'),
+        validation: [ Validators.required, regexValidator(/^\d+$/) ]
+    },
+    additional_params: {
+        placeholder : T('Additional parameters'),
+        tooltip: T('Define other sections and their respective key/value pairs (if any). Enclose each\
+ section name in square brackets, and put each key/value pair on a new line. Example: <br>\
+ [system.intr] <br> \
```

Comment:
Why include a space before the break here?

Reply:
space removed